R/tidyverse.R

```diff
@@ -251,6 +238,32 @@ use_tidy_style <- function(strict = TRUE) {
   invisible(styled)
 }
 
+#' Use tidy evaluation
```

Comment:
We do already have a `R/usethis-defunct.R` file for such things. I suggest we move `use_tidy_eval()` there.

Reply:
That currently yields one giant Rd file where it's hard to match old function with new function. Do you mind if I convert it to one Rd per function (or group of functions)?

Reply:
I'm fine with that. Of course, when it started, it wasn't giant. Then it gradually became so.